### ISSUE TITLE
Feedback on all tests regarding definition of the word "conveyed"

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria-at#1317]
ARIA-AT should define "convey" and link to that definition from the tests where that term is used.

From the test results, most inexperienced testers appear to assume "convey" means that the thing to be conveyed should be spoken with literal text-to-speech synthesis. However, there are cases where a sound icon is sufficient for hearing-but-blind screen reader users. Likewise, screen readers can 'convey' things through braille displays that are non-literal, too... Selection status, table layout, contextual iconography, tactile graphics on matrix displays, etc…

[This alert test](https://github.com/w3c/aria-at/issues/1316) is one example where lack of a clear definition caused a problem, but it's not the only instance... 

For example, even on a 100% passing test as simple as "Navigate forward to a link", the phrasing "role link is conveyed" may lead some inexperienced testers to assume the work "link" needs to be spoken audibly. But with non-default lower verbosity settings, the word "link" is dropped in favor of a sound icon for many roles conveyed by VoiceOver to blind users with hearing. 

Likewise with braille, there are different use cases for hybrid screen readers users (hearing+braille) than for braille-only users (usually both deaf and blind), but role is often conveyed either with a abbreviation ('btn' for button) or with some braille iconography. See [VoiceOver's Braille + RTT support article](https://support.apple.com/en-us/101784) for an example of animated braille iconography.

First draft example suggestion of a definition: "~Convey: to communicate by any means, whether by generated speech, brailled text, unique audible notifications (including sound icons), unique tactile indication (including symbolic braille cells, haptics, etc), or any other means that is perceivable and reasonably understandable to an experienced user of the screen reader." (Maybe add some perceivability limitations in specific use cases like deaf blind braille-only scenarios.)

By defining a more accurate and expansive version of the term, the test expectations could be more resilient perhaps even with non-default SR settings.
